docs/modules/ROOT/pages/how-tos/restore_postgresql_burp.adoc

```diff
@@ -0,0 +1,56 @@
+= Restore PostgreSQL with BURP
+
+== Backup
+
+Dump destination directory: `/var/lib/postgresql-backup`
+
+The postgresql-pre-backup script uses https://www.postgresql.org/docs/current/app-pgdump.html[pg_dump]
+to export all PostgreSQL databases in a consistent data format before every BURP run, usually once a day.
+The destination files are named after the database name.
+
+The default dump format used is "custom", it may be overriden by DUMP_FORMAT in `/etc/default/postgresql-pre-backup`
+
+Example file list:
+
+[source,bash]
+--
+confluence.dump
+postgres.dump
+--
+
+== Restore
+
+Backups can be re-imported with the pg_restore command. The following commands should be seen as examples.
+An actual restore procedure depends on the situation at hand (partial restore after accidental data removal or a full restore after catastrophic hardware failure).
+
+NOTE: This will only restore data, but not roles. Roles are supposed to be configured via puppet/hieradata.
+
+. Restore from BURP (see xref:how-tos/restore_from_backup.adoc[Restore Files from Backup] for details and more options):
++
+[source,bash]
+--
+$ sudo burp -a r -d /tmp/restore -r ^/var/lib/postgresql-backup/
+$ sudo chown -R postgres:postgres /tmp/restore
+--
+
+NOTE: If the service runs on an encrypted disk, ensure you use a restore location
+that is on an encrypted disk as well.
+
+. Load backup (potentially destructive!):
++
+[source,bash]
+--
+$ sudo -u postgres pg_restore ~/restore/db.dump -d db
```

Comment:
wouldn't it be a better approach to rename the old db first? 

```
ALTER DATABASE db RENAME TO old_db;
```

Reply:
I don't know, it really depends on the use case. We could add it as optional.

It is the same with ANALYZE, in general in postgres it is recommended to run ANALYZE on the database to warm the cache for the query planner, but it can also prolong the whole restore process. Not sure where to draw the line here with the "simple restore" instructions.

Reply:
added something in [4647bfe](https://github.com/vshn/managed-service-docs/pull/6/commits/4647bfe519aa7db90e6ae8cd9d21a94d8e4236e9)